app.js

```diff
@@ -45,7 +45,7 @@ app.use(function(req, res, next){
 });
 app.use(authRoutes);
 app.use("/campgrounds",campgroundRoutes);
-app.use(commentRoutes);
+app.use("/campgrounds/:id/comments",commentRoutes);

```

Comment:
not a super fan of this, you could have problems in the future if you continue adding routes to `commentRoutes`. But ... it's fine